src/settings.js

```diff
@@ -70,6 +70,19 @@ export class SettingsStore {
     return undefined;
   }
 
+  /**
+   *
+   * @param {Record<string, string>} item the item to serialize
+   * @returns {Record<string,AttributeValue>} the serialized item
+   */
+  static serializeItem(item) {
+    const newItem = {};
+    Object.keys(item).forEach((k) => {
+      newItem[k] = { S: item[k] };
```

Comment:
what is `S`?
what about using `clone`?

Reply:
It’s a super jenky construct used for typing in the DynamoDB SDK. It’s not the world’s friendlies SDK to work with